pycaret/tests/test_time_series.py

```diff
@@ -358,3 +358,55 @@ def test_tune_model_raises(load_data):
         exceptionmsg
         == f"`search_algorithm` must be one of 'None, random, grid'. You passed '{search_algorithm}'."
     )
+
+
+def test_predict_model_customization(load_data):
+    """Tests predict model customization"""
+
+    exp = TimeSeriesExperiment()
+
+    fh = 12
+    fold = 2
+    data = load_data
+
+    exp.setup(
+        data=data,
+        fh=fh,
+        fold=fold,
+        fold_strategy="expanding",
+        verbose=False,
+        session_id=42,
+    )
+
+    model = exp.create_model("arima")
+
+    expected_period_index = load_data.iloc[-fh:].index
+
+    # Default prediction
```

Comment:
is there a way to parametrize these test cases to report each individual case in pytest report ?

Reply:
Good point. I combined it with the `create_model` test so that it tests it for all models.